tasks/frontEnd/src/assets/base.css

```diff
@@ -0,0 +1,86 @@
+/* color palette from <https://github.com/vuejs/theme> */
+:root {
```

Comment:
Only because utilising the inbuilt styles from PolarisVue seemed to create a UI that was usable for both desktop and mobile without further manipulation